batch/batch/front_end/front_end.py

```diff
@@ -1626,10 +1629,52 @@ def plot_job_durations(container_statuses: dict, batch_id: int, job_id: int):
     return json.dumps(fig, cls=plotly.utils.PlotlyJSONEncoder)
 
 
+
+
 def plot_resource_usage(
         resource_usage: Optional[Dict[str, Optional[pd.DataFrame]]]
 ) -> Optional[str]:  # pylint: disable=unused-argument
-    return None
+    fig = make_subplots(
+        rows=2, cols=1,
```

Comment:
You might need to reformat this file with black. See what CI says in `check_services`.

Reply:
I ran it and there were no issues